### PR TITLE
 ✨ Add file probe service

### PR DESCRIPTION
**Pull request by wvangeit**
## What do these changes do?

Adds a file probe, where a service can write files to. It will then show up in the outputs of a job running the study. (requirement for the metamodelling workflow)

## Related issue/s
https://github.com/ITISFoundation/osparc-issues/issues/1360

## How to test

Create a new output of a service, the file probe is a available as an output

## Dev-ops checklist

- [ x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Comment by wvangeit** [comment]
Added @odeimaiz since it's semi-frontend, and although it's not a requirement for now, it would be good to in the end at a button etc to this service to download the file etc.

File: packages/models-library/src/models_library/function_services_catalog/services/probes.py [hunk 1]
```diff
@@ -66,7 +66,30 @@ def create_metadata(type_name: str, prefix: str | None = None) -> ServiceDockerD
...
+    {
+        "integration-version": LATEST_INTEGRATION_VERSION,
+        "key": f"{FUNCTION_SERVICE_KEY_PREFIX}/iterator-consumer/probe/file",
+        "version": "1.0.0",
+        "type": ServiceType.FRONTEND,
+        "name": "File probe",
+        "description": "Probes its input for files",
+        "thumbnail": create_fake_thumbnail_url("file"),
+        "authors": [
+            OM,
```
Comment (bisgaard-itis) [inline packages/models-library/src/models_library/function_services_catalog/services/probes.py:79]: Maybe WVG?

File: packages/models-library/src/models_library/function_services_catalog/services/probes.py [hunk 2]
```diff
@@ -66,7 +66,30 @@ def create_metadata(type_name: str, prefix: str | None = None) -> ServiceDockerD
...
+        ],
+        "contact": OM.email,
```
Comment (bisgaard-itis) [inline packages/models-library/src/models_library/function_services_catalog/services/probes.py:81]: Here too

**Review by bisgaard-itis — APPROVED** [review]
Very cool, looks good. Thanks a lot. 
I think it would make sense to also reference 
- https://github.com/ITISFoundation/osparc-issues/issues/1343

in the PR description. 
Side note: A link to an issue displays much nicer if you put it in a bullet point in Github.

File: packages/models-library/src/models_library/function_services_catalog/services/probes.py (hunk 1 above)
Comment (wvangeit) [reply packages/models-library/src/models_library/function_services_catalog/services/probes.py:79]: Ah ok, was not sure if it was referring to a real author, or that it meant 'osparc maintenaner'. Do I need to add a key with WVG somewhere in another file in the repo?

Comment (wvangeit) [reply packages/models-library/src/models_library/function_services_catalog/services/probes.py:79]: It seems to be pointing some environment variables after a couple indirections, but then i don't see anymore how these are set. @odeimaiz , you know by chance?

File: packages/models-library/src/models_library/function_services_catalog/services/probes.py (hunk 2 above)
Comment (wvangeit) [reply packages/models-library/src/models_library/function_services_catalog/services/probes.py]: Changed to WVG

**Review by matusdrobuliak66 — APPROVED** [review]

**Comment by pcrespov** [comment]
> @pcrespov @odeimaiz I didn't really find tests for these meta-services, but maybe I overlooked them. If you know if there are any, I can add a test similar to these.

test is automatic. the data that you have added is validation upon import

**Review by pcrespov — APPROVED** [review]

**Review by odeimaiz — APPROVED** [review]

**Comment by wvangeit** [comment]
> > @pcrespov @odeimaiz I didn't really find tests for these meta-services, but maybe I overlooked them. If you know if there are any, I can add a test similar to these.
> 
> test is automatic. the data that you have added is validation upon import

I like automatic tests ;-)

**Review by sanderegg — APPROVED** [review]

**Pull request merged**